config/routes.rb

```diff
@@ -3,13 +3,21 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  get 'comments/create'
+  root "users#index"
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show, :new, :create, :destroy] do
       resources :comments, only: [:index, :show, :new, :create, :destroy]
       resources :likes, only: [:create]
     end
   end
-  root "users#index"
 
+  namespace :api, defaults: { format: :json } do
+    namespace :v1 do
+      resources :users, only: [:index, :show] do
+        resources :posts, only: [:index, :show] do
+          resources :comments, only: [:index, :create]
+        end
```

Comment:
- [ ] Based on the project requirements, there needs to be an API endpoint that retrieves all comments for a post and posts comments on a post. The endpoint should be accessed using a GET and POST request to `/api/v1/users/user_id/posts/post_id/comments`. However, when I attempted to access this endpoint to get comments and post comments, I got this error message below :point_down: . Please make sure that you fix this issue.

  
  ![Screenshot from 2023-07-29 18-43-30](https://github.com/ShiroYaksha90/blog-app/assets/92303891/895bbaf7-2b4b-49d9-8a9f-51c0a04b7bd1)
  
  ![image](https://github.com/ShiroYaksha90/blog-app/assets/92303891/20a3b720-dea0-4e22-bf84-9430c3b903ca)

Reply:
This change has been done well